src/main/kotlin/minesweeper/domain/Cells.kt

```diff
@@ -0,0 +1,40 @@
+package minesweeper.domain
+
+class Cells(val cells: List<Cell>, val width: Int, val height: Int) {
+
+    init {
+        require(width > 0 && height > 0) { "너비와 높이는 0보다 커야합니다" }
+        require(cells.size == width * height) { "셀의 크기는 너비와 높이를 곱한 값이어야 합니다" }
+    }
+
+    fun getCellValue(index: Int): Int {
+        return Position.values().filter { isExistMine(it, index) }.count()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return Position.getIndexesNearByLocation(getLocation(index), width, height).count { cells[it].isMine }
```